### PR TITLE
Release v1.0.0

### PR DESCRIPTION
**Pull request by benjaminwil**
What is the goal of this PR?
---

This pull request ties the knot for version 1.0.0 of this extension.

### CODE REVIEW

**Review by forkata — APPROVED** [review]

**Comment by benjaminwil** [comment]
Hi everyone, please don't merge this until #253 is merged.

**Review by Noah-Silvera — APPROVED** [review]

**Pull request merged**
